Mailbox/Person.cs

```diff
@@ -1,4 +1,50 @@
-﻿namespace Mailbox
+﻿using System;
+
+namespace Mailbox
 {
+    /*
+     * Create a new Person struct
+     * A person has a first and last name
+     * Person should implement IEquatable<Person>
+     * Ensure appropriate equality operators are also written
+     * Implement a reasonable ToString method
+     */
+    public struct Person : IEquatable<Person>
+    {
+        public string? FirstName;
+        public string? LastName;
+
+        public Person(string firstName, string lastName)
+        {
+            if (firstName is null)
+                throw new ArgumentNullException($"{nameof(firstName)} cannot be null.");
```

Comment:
First argument is just the parameter name, second is message.
```suggestion
                throw new ArgumentNullException(nameof(firstName));
```